src/components/partnership/partnership.service.ts

```diff
@@ -500,13 +593,20 @@ export class PartnershipService {
   }
 
   async update(input: UpdatePartnership, session: ISession) {
+    // mou start and end are now computed fields and do not get updated directly
```

Comment:
Which is why they aren't in the `UpdatePartnership` DTO 😉

Reply:
just helping other devs be alerted to changes they may not be aware of.